src/translators/crate.py

```diff
@@ -284,6 +288,22 @@ def _insert_entities_of_type(self,
         self.cursor.executemany(stmt, entries)
         return self.cursor
 
+    def _attr_is_structured(self, a):
+        # isinstance(a, object)
+        isdict = False
+
+        if a['value'] is not None and isinstance(a['value'], dict):
+            self.logger.info("attribute {} has 'value' attribute of type dict".format(a))
+            isdict = True
+        # the next test is probably unuseful
```

Comment:
i think we should not store it.

Reply:
I think it gets stored with a null value because of #125 , but should double check. 
I think if we've seen that attribute before, we should store it with a null value---again see discussion in #125. But I agree with you in the case we haven't seen that attribute yet. In other words, if an attribute A comes in with no value nor type---e.g. `A = { k: 123, value: null, type: ''}`---and this is the *first* time we save the enclosing entity, then we should exclude A from the list of attrs we stash away as opposed to creating a column A of type `string`. This way, if the next time around the entity does have an attribute A with a type which isn't text, we can still store it without problems....

Reply:
Zapped the comment, added a test case for it (`test_store_empty_attribute_as_null_text`) and opened an issue to review translation corner cases: #152.